charts/k8s-monitoring/values.yaml

```diff
@@ -2029,6 +2080,18 @@ opencost:
     nodeSelector:
       kubernetes.io/os: linux
 
+# Settings for the Kepler deployment
+# You can use this sections to make modifications to the Kepler deployment.
+# See https://github.com/sustainable-computing-io/kepler-helm-chart/tree/main/chart/kepler for available values.
+kepler:
+  # -- Should this Helm chart deploy Kepler to the cluster.
+  # Set this to false if your cluster already has Kepler, or if you do not want to scrape metrics from Kepler.
+  # @section -- Chart
+  enabled: false
+
+  # @ignored -- This skips including these values in README.md
+  canMount:
+    usrSrc: false
```

Comment:
Also consider adding below to `extraEnvVars:`

```
  EXPOSE_ESTIMATED_IDLE_POWER_METRICS: "true"
```
this was recommended from #kepler-project channel at CNCF slack